meteor-app/imports/startup/server/assessment-data.js

```diff
@@ -80,219 +80,381 @@ Meteor.methods({
     const parts = [
       {
         name: 'Front Tyre (second hand)',
-        price: 500
+        price: 500,
+        code: 'F',
+        category: 'tyre',
+        used: true
       },
       {
         name: 'Rear Tyre (second hand)',
-        price: 500
+        price: 500,
+        code: 'R',
+        category: 'tyre',
+        used: true
       },
       {
         name: 'Front Tyre (new)',
-        price: 1000
+        price: 1000,
+        code: 'F',
+        category: 'tyre',
+        used: false
       },
       {
         name: 'Rear Tyre (new)',
-        price: 1000
+        price: 1000,
+        code: 'R',
+        category: 'tyre',
+        used: false
       },
       {
         name: 'Brake pads - Dual pivot',
-        price: 1500
+        price: 1500,
+        code: 'O',
+        category: 'other',
+        used: false
       },
       {
         name: 'Brake pads - V brakes',
-        price: 1000
+        price: 1000,
+        code: 'O',
+        category: 'other',
+        used: false
       },
       {
         name: 'Tube',
-        price: 800
+        price: 800,
+        code: 'O',
+        category: 'other',
+        used: false
       },
       {
         name: 'Tube & fitting',
-        price: 2000
+        price: 2000,
+        code: 'O',
+        category: 'other',
+        used: false
       },
       {
         name: 'Cable x 1',
-        price: 500
+        price: 500,
+        code: 'O',
+        category: 'cable',
+        used: false
       },
       {
         name: 'Cables x 2',
-        price: 1000
+        price: 1000,
+        code: 'O',
+        category: 'cable',
+        used: false
       },
       {
         name: 'Cables x 3',
-        price: 1500
+        price: 1500,
+        code: 'O',
+        category: 'cable',
+        used: false
       },
       {
         name: 'Cables x 4',
-        price: 2000
+        price: 2000,
+        code: 'O',
+        category: 'cable',
+        used: false
       },
       {
         name: 'Cable fitted x 1',
```

Comment:
As we are introducing items for front and rear brake cables, we can get rid of these x1, x2 etc